api/api.go

```diff
@@ -242,6 +255,55 @@ func OrbPublish(ctx context.Context, logger *logger.Logger,
 	return &response.PublishOrb.PublishOrbResponse, err
 }
 
+// OrbPublishID publishes a new version of an orb by id
+func OrbPublishID(ctx context.Context, logger *logger.Logger,
```

Comment:
`OrbPublishByID` ?  `OrbPublishID` sounds like it's fetching and returning an `ID`.